sixtyfps_compiler/build.rs

```diff
@@ -23,11 +23,22 @@ fn widget_library() -> &'static [(&'static str, &'static BuiltinDirectory<'stati
 "#
     )?;
 
+    let mut known_styles = Vec::<String>::new();
     for style in read_dir(library_dir)?.filter_map(Result::ok) {
         let path = style.path();
         if !path.is_dir() {
             continue;
         }
+
+        let style = path
+            .file_name()
+            .expect("Readdir should always return an entry with a name")
+            .to_string_lossy()
+            .to_string();
+        if style != "common" {
```

Comment:
the actual test is if there is a `sixtyfps_widgets.60` file in it.

Reply:
Implemented.